frontend/apps/filemanager/filemanager.lua

```diff
@@ -357,8 +357,10 @@ function FileManager:init()
                 plugin_module, { ui = self, })
             -- Keep references to the modules which do not register into menu.
             if ok then
+                local name = plugin_module.name
+                if name then self[name] = plugin_or_err end
```

Comment:
If there is no issue inserting it with a name now, instead of just having it have an index (table.insert), looks alright.

Reply:
It's the same thing ReaderUI does:

https://github.com/koreader/koreader/blob/39be43ad8441a8f5c2508afba261444672144dc3/frontend/apps/reader/readerui.lua#L76-L80

The table.insert index is used for internal actions like event propagation (which runs on `ipair()`), while the named index is used for our convenience.